.github/workflows/node.yml

```diff
@@ -128,39 +195,40 @@ jobs:
                   cargo-toml-folder: ./node/rust-client
               name: lint node rust
 
-    # build-macos-latest:
-    #     runs-on: macos-latest
-    #     timeout-minutes: 25
-    #     steps:
-    #         - uses: actions/checkout@v4
-    #           with:
-    #               submodules: recursive
-    #         - name: Set up Homebrew
-    #           uses: Homebrew/actions/setup-homebrew@master
-
-    #         - name: Install NodeJS
-    #           run: |
-    #               brew update
-    #               brew upgrade || true
-    #               brew install node
-
-    #         - name: Downgrade npm major version to 8
-    #           run: |
-    #               npm i -g npm@8
-
-    #         - name: Build Node wrapper
-    #           uses: ./.github/workflows/build-node-wrapper
-    #           with:
-    #               os: "macos"
-    #               named_os: "darwin"
-    #               arch: "arm64"
-    #               target: "aarch64-apple-darwin"
-    #               github-token: ${{ secrets.GITHUB_TOKEN }}
-    #               engine-version: "7.2.5"
+    build-macos-latest:
```

Comment:
```suggestion
    build-macos:
```

Reply:
Is there a chance to merge this job with one above? I mean to merge `test-ubuntu` and `test-macos` and use `runs-on` from matrix - like we do this for java and python clients

Reply:
They are very different, that's why it's also Ubuntu latest